### PR TITLE
Tutorial char enc

### PR DESCRIPTION
**Pull request by brandelune**

### CODE REVIEW

File: tutorials/language-decl/index.fr.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <html lang="fr">
 <head>
 <meta charset="utf-8" />
-<title>Travailler avec les langues en HTML</title>
-<meta name="description" content="Tutoriel W3C i18n : Comment déclarer et utiliser les langues dans les fichiers XHTML et HTML" />
-<script type="application/javascript">
+<title>Travailler avec les langues en HTML (tutoriel)</title>
+<meta name="description" content="Tutoriel W3C i18n : Comment déclarer et utiliser les langues dans les fichiers XHTML et HTML" />
```
Comment (r12a) [inline tutorials/language-decl/index.fr.html:6]: Looks like you're using nbsp.  The recent trend is to use U+202F NARROW NO-BREAK SPACE for French punctuation.  What do you think?

File: tutorials/language-decl/index.fr.html [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+<script>
```
Comment (r12a) [inline tutorials/language-decl/index.fr.html:7]: Well spotted!

File: tutorials/language-decl/index.fr.html [hunk 3]
```diff
@@ -39,25 +39,24 @@
...
 <nav id="mainNavigation"></nav><script>document.getElementById('mainNavigation').innerHTML = mainNavigation</script>
 
 <h1>Travailler avec les langues en HTML (tutoriel)</h1>
 </header>
 
-
 <div id="sidebarExtras">
-  <section>
-    <h2 id="whyread" class="notoc">Pourquoi devriez-vous lire ces articles?</h2>
+	<section>
```
Comment (r12a) [inline tutorials/language-decl/index.fr.html:51]: Hmm. Another thing i missed while updating the templates: the id should always be on the section tag.  Would you mind fixing it for the French (i'll fix the other languages).

File: tutorials/language-decl/index.fr.html [hunk 4]
```diff
@@ -66,45 +65,51 @@ <h2 class="notoc">Objectifs</h2>
...
-  <div class="example">
-  <p><code translate="no">&lt;html lang=&quot;fr&quot;&gt;</code></p>
-  </div>
-		    <p>S’il s’agit d’un document polyglotte XHTML 1.x ou HTML5 servant de XML, vous devriez également utiliser l’attribut <code class="kw" translate="no">xml:lang</code> (avec la même valeur). Si votre page sert uniquement de XML, utilisez uniquement l’attribut <code class="kw" translate="no">xml:lang</code>.</p>
-	      <p>N’utilisez pas les éléments <code class="kw" translate="no">meta</code> avec <code class="kw" translate="no">http-equiv</code> réglés sur <code class="kw" translate="no">Content-Language</code>.</p>
-	      <p>Choisissez les balises dans le registre des sous-balises de langues de l’IANA, et suivez les règles de syntaxe dans BCP 47 lorsque vous utilisez plus d’une sous-balise de langue. Veuillez garder vos balises de langue aussi courtes que possible.</p>
-		    <p>Utiliser la propriété <code class="kw" translate="no">:lang</code> dans CSS pour appliquer des styles en fonction de la langue de votre contenu. </p>
+<h2 id="quicksummary">En bref</h2>
```
Comment (r12a) [inline tutorials/language-decl/index.fr.html:79]: (This quicksummary id is ok on the h2 tag.)

File: tutorials/tutorial-char-enc/index.fr.html [hunk 5]
```diff
@@ -94,9 +93,9 @@ <h2>En bref</h2>
   <h2>Informations contextuelles essentielles</h2>
   <p>Si vous n’êtes pas familier à ce sujet, il vous sera nécessaire de comprendre un certain nombre de concepts fondamentaux afin de suivre les différentes parties du tutoriel. Si ces concepts vous sont familiers, vous pouvez <a href="#Slide0110">passer à la section suivante</a>.</p>
   <ul>
-    <li><a class="print" href="/International/questions/qa-what-is-encoding">Les encodages de caractères pour débutants</a></li>
+    <li><a class="print" href="/International/questions/qa-what-is-encoding.fr">Les encodages de caractères pour débutants</a></li>
```
Comment (r12a) [inline tutorials/tutorial-char-enc/index.fr.html]: No .fr extensions, please! The links are content negotiated.  This applies to several edits. Please change them back, thanks.

File: tutorials/tutorial-char-enc/index.fr.html [hunk 6]
```diff
@@ -158,12 +166,12 @@ <h2>Pour en savoir plus</h2>
...
 	  <li>
-	    <p>Vous commencez ? <a href="/International/getting-started/characters"><cite>Introduction aux jeux et encodages de caractères</cite></a></p>
+	    <p>Vous débutez ? <a href="/International/getting-started/characters.fr"><cite>Présentation des jeux et encodages de caractères</cite></a></p>
       </li>
 	<li>
-      <p><cite>Créer du contenu HTML et CSS</cite></p>
+      <p><cite>Création de contenus HTML et CSS</cite></p>
       <ul>
-        <li><a href="/International/techniques/authoring-html#charset">Caractères</a> [anglais]</li>
+        <li><a href="https://www.w3.org/International/techniques/authoring-html.en?open=charset#charset">Caractères</a></li>
```
Comment (r12a) [inline tutorials/tutorial-char-enc/index.fr.html:174]: We should drop the "https://www.w3.org" and the ".en".

**Review by r12a — CHANGES_REQUESTED** [review]

File: tutorials/language-decl/index.fr.html (hunk 1 above)
Comment (brandelune) [reply tutorials/language-decl/index.fr.html:6]: It's not trivial to type even on a mac keyboard so I usually use NBSP. But I'll batch replace the thing and will commit again.

File: tutorials/language-decl/index.fr.html (hunk 2 above)
Comment (brandelune) [reply tutorials/language-decl/index.fr.html:7]: OmegaT does that automatically. It takes the structure of the original and will just fill it with the translated texts. So you have to thank OmegaT :-)

File: tutorials/language-decl/index.fr.html (hunk 3 above)
Comment (brandelune) [reply tutorials/language-decl/index.fr.html:51]: ok

File: tutorials/language-decl/index.fr.html (hunk 4 above)
Comment (brandelune) [reply tutorials/language-decl/index.fr.html:79]: Nothing to do then ?

File: tutorials/tutorial-char-enc/index.fr.html (hunk 5 above)
Comment (brandelune) [reply tutorials/tutorial-char-enc/index.fr.html]: I'm on it.

**Pull request closed without merging**